docs/sources/tempo/operations/grafana_datasource.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Grafana Tempo Datasource
+description: A collection of documents that detail how to deploy tempo and use it with Grafana
+aliases:
+ - /docs/tempo/latest/gateway
+ - /docs/tempo/operations/gateway
+weight: 15
+---
+
+# Grafana Tempo Datasource
+
+There are one main thing to take into account when you deploy the `TempoStack` and want to use as Grafana datasource.
+
+If the tempo is deployed the Gateway in front, an authentication information needs to be provided to Grafana, and the URL of the tenant it wants to access.
+
+If the gateway is not used, the only requirement is to Grafana can access to the `frontend-query` endpoints.
```

Comment:
I think the first few sentences in this page are really discussing additional things a user needs to have or do. 

The intro uses `frontend-query` and the Without the gateway section uses `query-frontend`. Which one is correct? 

```suggestion
# Grafana Tempo Datasource

Using the instructions on this page, you can configure the `TempoStack` to send data to Grafana and configure the Tempo data source.  

You can choose to either use the gateway or not use the gateway: 

* If the `TempoStack` is deployed using the Gateway in front, you'll need to provide authentication information  to Grafana, along with the URL of the tenant it wants to access.

* If the gateway is not used, then you need to make sure Grafana can access to the `frontend-query` endpoints.
```

Reply:
query-frontend should be the right one